factory/django.py

```diff
@@ -25,6 +25,7 @@
 DEFAULT_DB_ALIAS = 'default'  # Same as django.db.DEFAULT_DB_ALIAS
 
 DJANGO_22 = Version(django_version) < Version('3.0')
+DJANGO_32 = Version('3.0') < Version(django_version) < Version('4.0')
```

Comment:
Why is this `DJANGO_32` and not `DJANGO_30`?
```suggestion
DJANGO_30 = Version('3.0') < Version(django_version) < Version('4.0')
```

Reply:
3.0 doesn’t exist anymore :/ and i wanted to exclude 2.2, and be under 4.0

maybe a `DJANGO_GT_22_LE_40` but that’s a mouthful

Reply:
3.2 is the only supported version of the 3.x series, I think that’s fine. Maybe `DJANGO_3`?

Reply:
True, but you haven’t removed 2.2 from the tests, figured it was still supported by this project. Actually, didn’t want to say anything since we’re still on 2.2 and would like to use this ASAP

damn, cats out of the bag, you’re going to have me remove 2.2 aren’t you? or worst yet someone is going to create a PR after reading this…

Reply:
@francoisfreitag please take a look at https://github.com/FactoryBoy/factory_boy/pull/931 — this pr is still in progress. I still need to clean the tests and write docs, the code may need some polishing.. look at the StepBuilder, now sure how to handle it. sigh, yeah it’s a work in progress